remix.init/package.json

```diff
@@ -5,7 +5,9 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "commander": "^11.0.0",
-    "inquirer": "^8.2.2"
+    "@npmcli/package-json": "^5.0.0",
+    "commander": "^11.1.0",
+    "execa": "^8.0.1",
```

Comment:
This version of execa is ESM-only, and init scripts need to be cjs. `5.1.1` is the one used by `@remix-run/dev`, so that would be the best choice